common/saber_base.h

```diff
@@ -114,8 +114,11 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(FONT_DIRECTORY_NOT_FOUND)     \
     /* menu effects, triggered when you go from one menu entry to the next/previous */                             \
     DEFINE_EFFECT(MENU_CHANGE)                  \
-
-
+    /* Detonator  effects */                    \
+    DEFINE_EFFECT(BGNARM)                       \
+    DEFINE_EFFECT(ARMHUM)                       \
+    DEFINE_EFFECT(ENDARM)                       \
```

Comment:
Detonator doesn't use endarm, just OFF.

Reply:
If you're going to add EFFECT_* values, you're also going to need to add code that triggers them, otherwise they will do nothing.